src/reduced_order/pod_basis_types.cpp

```diff
@@ -0,0 +1,124 @@
+#include "pod_basis_types.h"
+
+namespace PHiLiP {
+namespace ProperOrthogonalDecomposition {
+
+template <int dim>
+SpecificPOD<dim>::SpecificPOD(std::shared_ptr<DGBase<dim,double>> &_dg)
```

Comment:
Same comment regarding the underscore notation

Reply:
Fixed in most recent commit.